fossid-webapp-client/src/main/kotlin/api/result/MatchType.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api.result
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+enum class MatchType {
+    @JsonProperty("none")
+    NONE,
+    @JsonProperty("Ignored")
```

Comment:
Is it really correct that this is the only capitalized property?

Reply:
yes
```
"created": "2020-09-28 08:49:53",
      "scan_id": "2910",
      "scan_file_id": "4165084",
      "file_id": "19283",
      "match_type": "Ignored",
```
and 
```
"scan_id": "2910",
      "scan_file_id": "4165407",
      "file_id": "527636",
      "match_type": "none",
```
for instance